openmc/universe.py

```diff
@@ -469,7 +469,7 @@ def get_nuclide_densities(self):
         """
         nuclides = OrderedDict()
 
-        if self._atoms is not None:
+        if len(self._atoms) > 0:
```

Comment:
Alternatively, you could use the "truthiness" of the object (`dict` is considered True if it is non-empty)
```suggestion
        if self._atoms:
```